<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <widget class="QComboBox" name="findLineEdit">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>11</y>
     <width>431</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="findNext">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="icon">
    <iconset theme="go-down"/>
   </property>
  </widget>
  <widget class="QPushButton" name="selectAll">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>272</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>14</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findPrevious">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>271</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="icon">
    <iconset theme="go-up"/>
   </property>
  </widget>
  <widget class="QCheckBox" name="advancedOptions">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>47</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show advanced options</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxAdvanced">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>79</y>
     <width>461</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Advanced</string>
   </property>
   <widget class="QRadioButton" name="findPlainText">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>32</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Find &amp;plain text</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="findRegularExpression">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>60</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Find with &amp;regular expressions</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="findSpecialCharachters">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>89</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Find &amp;with special characters (\n, \r, \t, \0, \u...,\x...)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="matchWholeWord">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>120</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Match whole word only</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="matchCase">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>150</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Match case</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
